README.md

```diff
@@ -54,20 +54,52 @@ The [Salt Contributing guide][salt-contributing] has a lot of relevant informati
     # 1. Make a local salt dir
     mkdir -p local/etc/salt
 
-    # 2. Create a minion config
+    # 2. Make a local dir for salt state files
+    mkdir -p local/srv/salt
+
+    # 3. Make a local dir for salt pillar files
+    mkdir -p local/srv/pillar
+
+    # 4. Create a minion config
     cat << EOF> local/etc/salt/minion
     user: $(whoami)
     root_dir: $PWD/local/
     file_root: $PWD/local
     master: localhost
     id: saltdev
     master_port: 55506
+    pillar_roots:
+      base:
+        - $PWD/local/srv/pillar
+    EOF
+
+    # 5. Make a Saltfile
+    cat << EOF> Saltfile
+    salt-call:
+      local: true
+      config_dir: local/etc/salt
+    EOF
+
+    # 6. Create a pillar file for you configuration
+    cat << EOF> local/srv/my_vsphere_conf.sls
+    # vCenter
+    vmware_config:
+      host: insert_host_name_or_ip_address
+      password: insert_password
+      user: insert_user_name
```

Comment:
**suggestion** Use TEST-NET IP or example.com domain. Also update to use our modern key.

```suggestion
    saltext.vmware:
        # Or use IP address, e.g. 203.0.113.42
        host: vsphere.example.com
        password: CorrectHorseBatteryStaple
        user: BobbyTables
```